---
layout: page
title: Research
description: Maya Lemmon-Kishi | Research 
---

<div style="text-align: justify">Some projects I have worked on during my PhD. Come back for exciting future updates on the progress and results of these projects!!</div>

### Molecular Dating of Sedimentary Ancient DNA

<div style="text-align: justify">Environmental DNA (eDNA) is a rich source of genetic data that captures a broad perspective of biodiversity. Ancient eDNA from lake sediments and permafrost can be recovered and employed to explore patterns of biodiversity across the globe over thousands of years. To discover ecologically relevant patterns, ancient eDNA samples must be accurately dated. While there are many techniques for sample dating, each method has its own limitations such as location-specific calibration and time range. Furthermore, the majority of these techniques are dating the environmental sample and not the DNA itself. The few existing molecular dating methods only use a small fraction of the sequence data, which can result in low confidence of the age estimate. I am developing ratePlacer, a phylogeny-based method for analyzing sedaDNA that can combine the information from many short reads in a sample while accounting for DNA damages to provide maximum likelihood estimates of sample ages.</div>

### Molecular Clock Calibration Using Sedimentary Ancient DNA

<div style="text-align: justify">In many species there is a great deal of uncertainty regarding the rate of substitution, implying that the molecular clock is poorly calibrated. Historically, molecular clock calibration has required species divergence times to be known from the fossil record or from geologically dated events, limiting the use of molecular clocks for many species, including bacteria. Sedimentary ancient DNA (sedaDNA) is a rich source of genetic data that captures broad taxonomic diversity over long time transects. These transects of ancient DNA across time can, in theory, be used to calibrate the molecular clock for a large variety of species, including bacteria. However, such analyses are challenged by the fragmented and damaged nature of short read ancient DNA. Here we can use ratePlacer to calibrate the molecular clock when sample ages are known. We plan to calibrate the molecular clokc of popular metabarcodes such as COI and 16s, but also of organelluar genomes across various taxonomic groups. Our new calibrations of the molecular clock will be useful for dating split times in phylogenies in many taxonomic groups that previously were lacking information about molecular clock rates.</div>

### Estimation of Phylogenetically Compatible Haplotypes from Environmental DNA for Population Genetic Analyses

<div style="text-align: justify">Currently eDNA analysis is limited to occupancy of species at sampling sites and is unable to explore the genetic structure of the populations detected. Exploring  eDNA data through population genetics theory will increase our understanding of ecological and evolutionary processes across spatiotemporal scales. Despite the insights this combination can provide, population genetic analyses are stymied by the fact that eDNA is composed of short reads from unknown numbers of individuals. This causes the haplotypes required for population genetic analyses to be fragmented, leading to a loss of information. I have been developing a method to jointly infer phylogenetically compatible haplotypes and sample frequencies using a penalized likelihood. By providing biologically relevant haplotypes and frequencies from eDNA, we will be able to explore shifts in biodiversity through the lens of population genetic theory.</div>

### Population Genetic Analyses for Environmental DNA

<div style="text-align: justify">During my rotation with the Nielsen lab, I explored the use of populaiton genetic summary statistics to analyze patterns of population struction from modern eDNA.</div>

---

<div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="https://github.com/lemmonquiche">GitHub</a></li>
          <li><a href="https://twitter.com/lemmonquiche">Twitter (@lemmonquiche)</a></li>
      </ul>
  </div>
</div>
