---
layout: page
title: Research
description: Aaron Stern | Research 
---

### Quantifying adaptation & correlated response in polygenic traits using whole-genome genealogies

We developed a method which uses state-of-the-art inference of ancestral recombination graphs (ARGs) to estimate the likelihood function of the selection gradient, giving us powerful tools to test traits for selection & correlated response to direct selection on other traits. We applied this method to UKBB GWASs and found evidence that traits such as hair color and educational attainment, are predominantly evolving under correlated response rather than direct selection. We also found antagonistic selection acting to increase T2D while decreasing HbA1c (a biomarker for hyperglycemia), suggesting that while complications of diabetes may be under negative selection, other aspects of T2D etiology (e.g. obesity) may have been under recent positive selection due to a changing environment (e.g., times of famine), supporting the thrifty gene hypothesis. Our results provide important context for the interpretation of tests for polygenic selection on correlated traits. 

<i>bioRxiv</i> (2020), <a href="https://doi.org/10.1101/2020.05.07.083402">available here</a>


### Inferring selection and frequency trajectories using ARGs

Most current methods for detecting natural selection from DNA sequence data do not make full use of the information available in DNA sequence data. We developed a new technique for approximating the full likelihood function for the selection coefficient, which can be used for detecting selection, estimating selection coefficients, and for inferring the allele frequency trajectory of a selected or neutral allele. We used the method to infer the past allele frequency trajectory for a lactase persistence SNP (<i>MCM6</i>) , as well as a set of 11 pigmentation-associated variants in Europeans. Several genes showed evidence of strong selection particularly within the last 5,000 years, including <i>ASIP</i>, <i>KITLG</i>, and <i>TYR</i>. However, selection on <i>OCA2/HERC2</i> seems to be much older and, in contrast to previous claims, we find no evidence of selection on <i>TYRP1</i>. 

<i>PLoS Genetics</i> (2019), <a href="https://doi.org/10.1371/journal.pgen.1008384">available here</a>

### In progress
Other projects I am currently pursuing include: the interaction between mutational load and adaptation, and implications for <b>genetic rescue</b>; reconstructing <b>selection on polygenic traits</b>; inferring heritability and polygenic selection in <b>structured populations</b>.

### Past work
#### Adaptation to diving in the Bajau people
In this work led by Melissa Ilardo, we found that Bajau ``Sea Nomads'' from the island of Sulawesi carry a genetic variant (PDE10A) associated with increased spleen size, regulating the blood oxygenation of these divers. I worked on modeling selection on PDE10A, showing that PDE10A has likely been under strong selection in the Bajau over the past 15,000 years. The work is available from <a href="https://doi.org/10.1016/j.cell.2018.03.054"><i>Cell</i></a>. 

#### Evolution of melanism in side-blotch lizards
In this work led by Ammon Corl, we found novel polymorphisms associated with darkened skin pigmentation in the side-blotch lizards of Pisgah, where lizard morphs are somewhat segregated by environment color (on-lava/light environment versus off-lava/dark environment). We found two genes, PREP and PRKAR1A, which are associated with pigmentation and show high divergence in Pisgah lizards. Using a similar approach to Ilardo <et al.> (2018), I showed that both PREP and PRKAR1A underwent strong selection, both mutations occurring within the last 500-1000 generations and rising to over 20% frequency in on-lava lizards. The work is available from <a href="https://doi.org/10.1016/j.cub.2018.06.075"><i>Current Biology</i></a>. 
